core/src/main/java/org/apache/iceberg/rest/HTTPClient.java

```diff
@@ -79,6 +82,13 @@ public class HTTPClient extends BaseHTTPClient {
   static final String REST_MAX_CONNECTIONS_PER_ROUTE = "rest.client.connections-per-route";
   static final int REST_MAX_CONNECTIONS_PER_ROUTE_DEFAULT = 100;
 
+  /** http proxy configuration for rest catalog */
```

Comment:
I'd rather remove this comment so that all of the properties are formatted and are next to each other